src/app/clusters/color-control-server/color-control-server.cpp

```diff
@@ -809,9 +818,14 @@ bool ColorControlServer::moveHueCommand(app::CommandHandler * commandObj, const
 
     // New command.  Need to stop any active transitions.
     stopAllColorTransitions(endpoint);
+    // now, kick off the state machine.
+    initHueTransitionState(endpoint, colorHueTransitionState, isEnhanced);
```

Comment:
I could do something like that. stopAllColorTransitions stop the shared timer for the transition calculation and updates. Doing the initXXTransitionState in stopAllColorTransitions is possible but requires extra parameters and some logic to know what state structure to initialize. 

However,I am not sure there is much gain to do it like that. I can see what it would look like

Reply:
If it's not worth doing, don't worry about it.